vision/google/cloud/vision/geometry.py

```diff
@@ -73,20 +87,34 @@ def __init__(self, latitude, longitude):
         self._longitude = longitude
 
     @classmethod
-    def from_api_repr(cls, response):
+    def from_api_repr(cls, location_info):
         """Factory: construct location information from Vision API response.
 
-        :type response: dict
-        :param response: Dictionary response of locations.
+        :type location_info: dict
+        :param location_info: Dictionary response of locations.
```

Comment:
Seems now to be just one location? Also, since you use the `latLng` key in your caller, this is now a `lat_lng` dict **not** a `location_info` dict.